src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb

```diff
@@ -114,7 +114,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shared Function ParseTrailingTrivia(text As String, Optional offset As Integer = 0) As SyntaxTriviaList
             Dim s = New InternalSyntax.Scanner(MakeSourceText(text, offset), VisualBasicParseOptions.Default)
             Using s
-                Return New SyntaxTriviaList(Nothing, s.ScanMultilineTrivia().Node, 0, 0)
+                Return New SyntaxTriviaList(Nothing, s.ScanSingleLineTrivia().Node, 0, 0)
```

Comment:
>ScanSingleLineTrivia [](start = 55, length = 20)

Is this a change in behavior if the following line is a doc comment or a directive?

Reply:
Yes.  But that's intentional. This is the ParseTrailingTrivia helper. Those are not trailing trivia.

Reply:
Please consider adding tests for those cases so the intent is clear.

---
In reply to: [390483174](https://github.com/dotnet/roslyn/pull/42234#discussion_r390483174) [](ancestors = 390483174)

Reply:
tests added!